episode-11/chunk.py

```diff
@@ -35,30 +34,37 @@ def __init__(self, world, chunk_position):
 
 		# mesh variables
 
-		self.mesh_vertex_positions = []
-		self.mesh_tex_coords = []
-		self.mesh_shading_values = []
+		self.mesh = []
+		self.translucent_mesh = []
 
-		self.mesh_index_counter = 0
-		self.mesh_indices = []
+		self.mesh_quad_count = 0
+		self.translucent_quad_count = 0
 
 		# create VAO and VBO's
 
 		self.vao = gl.GLuint(0)
-		gl.glGenVertexArrays(1, self.vao)
+		gl.glGenVertexArrays(1, ctypes.byref(self.vao))
```

Comment:
Any specific reason for this as opposed to just passing `self.vao`?

Reply:
I feel safer with pointer :troll: